policyengine_us/variables/gov/states/wv/tax/income/subtractions/wv_public_pension_subtraction.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class wv_public_pension_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia public pension subtraction"
+    defined_for = StateCode.WV
+    unit = USD
+    definition_period = YEAR
+    reference = "https://code.wvlegislature.gov/11-21-12/"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        public_pension_income = person("taxable_public_pension_income", period)
```

Comment:
Create a new `taxable_federal_pension_income` variable to capture this
```suggestion
        # "benefits received under any federal retirement system to which Title 4 U.S.C. §111 applies"
        # https://www.law.cornell.edu/uscode/text/4/111
        federal_pension_income = person("taxable_federal_pension_income", period)
```